ui/src/app/pages/analysis/interactive-notebook.tsx

```diff
@@ -59,7 +59,7 @@ const styles = reactStyles({
     border: 0
   },
   previewFrame: {
-    width: '100%',
+    width: 'calc(100% - 45px)',
```

Comment:
The scrollbar width is unlikely to be consistent across browser / OS, but this may look fine anyways. If you can push up a demo server I can check how it looks on Linux / Windows